packages/webapp/pages/_app.tsx

```diff
@@ -220,7 +222,7 @@ function InternalApp({ Component, pageProps, router }: AppProps): ReactElement {
       <DndContextProvider>
         {getLayout(<Component {...pageProps} />, pageProps, layoutProps)}
       </DndContextProvider>
-      {showBanner && (
+      {showBanner && !isFunnel && (
```

Comment:
Why do we not show it here when we could've probably used it so we won't have to introduce a new one.

Reply:
I know, but design is different and the component was already done :(